dist-material/release-docs/LICENSE.tpl

```diff
@@ -0,0 +1,33 @@
+{{ .LicenseContent }}
+
+=======================================================================
+Apache SkyWalking Subcomponents:
+
+The Apache SkyWalking project contains subcomponents with separate copyright
+notices and license terms. Your use of the source code for the these
+subcomponents is subject to the terms and conditions of the following
+licenses.
+========================================================================
+
+{{ range .Groups }}
+========================================================================
+{{ .LicenseID }} licenses
```

Comment:
One question, what is the sequence of this license and repo name? Alphabeta?

Reply:
They are ordered by dependency name.